docs/Deploying-Pantheon/Migration-Docker.md

```diff
@@ -36,10 +36,17 @@ was not available and the node key was always read from the data path.
 All file options (for example, [`--config-file`](../Reference/Pantheon-CLI-Syntax.md#config-file)) no longer 
 have a default. Add the relevant command line options to your Pantheon command line and specify the file path. 
 
-The [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) default is now `/opt/pantheon/database`. 
+The [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) default is now `/opt/pantheon`. 
 
 The [`--node-private-key-file`](../Reference/Pantheon-CLI-Syntax.md#node-private-key-file) default is 
-now `/opt/pantheon/database/key`. 
+now `/opt/pantheon/key`. 
+
+!!! important 
+    Do not mount a volume at the default data path (`/opt/pantheon`). Mounting a volume at the default 
+    data path stops the image from running. 
```

Comment:
Might be better worded as "...path will interfere with Pantheon's operation and prevent it from safely launching."

Reply:
Fixed